ci/docker/docker-compose.yml

```diff
@@ -0,0 +1,215 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We shouldn't need to bring this file into v1.x, since we aren't using docker-compose.